### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/scrapy/actions/runs/3727974993

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/scrapy/actions/runs/3727977653

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: docs/news.rst [hunk 1]
```diff
@@ -4700,7 +4700,7 @@ Scrapy 0.22.1 (released 2014-02-08)
 - BaseSgmlLinkExtractor: Added unit test of a link with an inner tag (:commit:`c1cb418`)
 - BaseSgmlLinkExtractor: Fixed unknown_endtag() so that it only set current_link=None when the end tag match the opening tag (:commit:`7e4d627`)
 - Fix tests for Travis-CI build (:commit:`76c7e20`)
-- replace unencodable codepoints with html entities. fixes #562 and #285 (:commit:`5f87b17`)
+- replace unencodeable codepoints with html entities. fixes #562 and #285 (:commit:`5f87b17`)
```
Comment (jsoref) [inline docs/news.rst:4703]: some projects don't like changing changelogs.

I'm trying to be fairly conservative about my changes..

File: sep/sep-018.rst [hunk 2]
```diff
@@ -594,18 +594,18 @@ A middleware to Scrape data using Parsley as described in UsingParsley
...
+                   for name in parsley.keys():
                        self.fields[name] = Field()
                super(ParsleyItem, self).__init__(*a, **kw)
            self.item_class = ParsleyItem    
-           self.parsley = PyParsley(parslet, output='python') 
+           self.parsley = PyParsley(parsley, output='python') 
 
        def process_response(self, response, request, spider):
-           return self.item_class(self.parsly.parse(string=response.body))
+           return self.item_class(self.parsley.parse(string=response.body))
```
Comment (jsoref) [inline sep/sep-018.rst:608]: notable...

File: tests/test_commands.py [hunk 3]
```diff
@@ -316,7 +316,7 @@ def _make_read_only(path: Path):
         self.assertEqual(actual_permissions, expected_permissions)
 
     def test_startproject_permissions_unchanged_in_destination(self):
-        """Check that pre-existing folders and files in the destination folder
+        """Check that preexisting folders and files in the destination folder
```
Comment (Gallaecio) [inline tests/test_commands.py:319]: This one seems arguable, but no strong opinion either way.

Same applies to “nonexistent” later on.

**Review by Gallaecio — APPROVED** [review]

**Comment by Gallaecio** [comment]
Closing and reopening to re-trigger CI jobs…

**Review by wRAR — APPROVED** [review]

**Pull request merged**